RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ChromeCacheExtractor.java

```diff
@@ -592,8 +592,13 @@ private void findExternalFiles(String cachePath) throws TskCoreException {
        
         // see if it is cached
         String fileTableKey = cacheFolderName + cacheFileName;
-        if (cacheFileName.startsWith("f_") && externalFilesTable.containsKey(fileTableKey)) {
-            return Optional.of(externalFilesTable.get(fileTableKey));
+
+        if (cacheFileName != null) {
+            if (cacheFileName.startsWith("f_") && externalFilesTable.containsKey(fileTableKey)) {
```

Comment:
@markmckinnon Why are the file names specific to those that start with "f_"? Is the idea to detect carved files? If so, our other carver creates carved files that have different naming structure.

Reply:
@eugene7646 the f_* files are extenal files based on the documentation in the code that are used to hold cache information.